Source/ProgressionSystemRuntime/Public/Data/PSDataAsset.h

```diff
@@ -105,4 +109,8 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSDataAsset : public UDataAsset
 	/** The Progression difficulty multiplier. */
 	UPROPERTY(EditDefaultsOnly, BlueprintReadOnly, meta = (BlueprintProtected, DisplayName = "Progression Multiplier", ShowOnlyInnerProperties))
 	TMap<EGameDifficulty, float> ProgressionDifficultyMultiplierInternal;
+
+	/** Stores the duration of fade-in/fade-out overlay animation in the main menu when cinematic started */
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Transient, Category = "C++", meta = (BlueprintProtected, DisplayName = "Fade duration"))
```

Comment:
- Data asset property can not be `Transient`, it's serious mistake preventing designers from changing default value of this property. **Please remove `Transient` specifier from this class-default property**

Reply:
removed: https://github.com/h4rdmol/ProgressionSystem/pull/20/commits/0ce10a7f2be61437f26c0d4feadca306d2594bb7